analyzer_test.go

```diff
@@ -49,17 +49,76 @@ func TestDeriveManifestAndLock(t *testing.T) {
 	}
 }
 
-func TestDeriveManifestAndLockDoesNotExist(t *testing.T) {
-	dir, err := ioutil.TempDir("", "dep")
+func TestAnalyzerDeriveManifestAndLockDoesNotExist(t *testing.T) {
+	h := test.NewHelper(t)
+	defer h.Cleanup()
+
+	h.TempDir("dep")
+
+	a := Analyzer{}
+
+	m, l, err := a.DeriveManifestAndLock(h.Path("dep"), "my/fake/project")
+	if m != nil || l != nil || err != nil {
+		t.Fatalf("expected manifest & lock & err to be nil: m -> %#v l -> %#v err-> %#v", m, l, err)
+	}
+}
+
+func TestAnalyzerDeriveManifestAndLockCannotOpen(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		// XXX: setting permissions works differently in
```

Comment:
Hehe... it's one of the strings that vim highlights inside comments, and it's commonly used to denote something that needs attention (FIXME, TODO, ...). dep and gps seem to have a mix of TODO and FIXME, with a few NOTEs here and there.

I'll change it to TODO. Hopefully someone more familiar with Windows spots that and figures how to do this there.